### ISSUE TITLE
kPhonetic 1558

### ISSUE THREAD

**Issue by paulmasson** [unicode-org/unihan-database#30]
This group currently has 42 characters and 34 additions. The additions all appear to be accurate, as can be seen with this link:

[https://analyticphysics.com/Language/Chinese Phonetic Groups.htm?current=1558&highlight](https://analyticphysics.com/Language/Chinese%20Phonetic%20Groups.htm?current=1558&highlight)

Please close this issue to signify agreement.

**Comment by kenlunde** [comment]
A couple of the additions seem suspicious to me, such as U+5F0E 弎 and U+38A5 㢥, which may make better additions to other phonetic groups, such as `1100*` and `1407*`, respectively. I am suspicious of a few others, too.

**Comment by paulmasson** [comment]
@kenlunde right you are! I had forgotten that U+5F0B 弋 is a radical. I'm beginning to see that many of the errors that occurred are because characters were added according to radical, rather than sound. See for example phonetic 964: many of the additions there belong elsewhere, but were added to this group for the cow radical.

In the spirit of Casey, when half of the character is not a radical then that part should take precedence in assignment. When both parts of the character are radicals, then I propose that we assign phonetics by whichever radical is closest in sound to the character.

I've created a pull request for the two characters you identified and four others. Let me know what you think.
